connect/resource/automation.py

```diff
@@ -41,17 +41,17 @@ def process_request(self, request):
     @function_log
     def approve(self, pk, data):
         # type: (str, dict) -> str
-        return self.api.post(path=pk + '/approve/', json=data)
```

Comment:
What is the idea under api -> client rename?

Reply:
The idea is to expose the REST client that the SDK uses internally to the user of the SDK, wrapped in an object that fulfills the logging requirements that currently are a work in progress. The "api" name of the property made more sense when this property was used to communicate with Connect API, but when exposing it as a client so we can use it for communication with vendor platforms also it seemed to me that it would make sense to rename it.

Anyway, since this feature is in development, I think probably it will be a better idea to rename the property to "_api" by now, to suggest private usage, and discuss this at a future time.

Reply:
ok, now it is clear